docs/nifgen/class.rst

```diff
@@ -5788,30 +5732,33 @@ delete_named_waveform
 
                 .. code:: python
 
-                    session.channels['0,1'].delete_named_waveform(waveform_name)
+                    session.channels['0,1'].delete_script(script_name)
 
 
-            :param waveform_name:
+            :param script_name:
 
 
-                Specifies the name to associate with the allocated waveform.
+                Specifies the name of the script you want to delete. The script name
+                appears in the text of the script following the script keyword.
 
                 
 
 
-            :type waveform_name: str
+            :type script_name: str
 
-delete_script
-~~~~~~~~~~~~~
+delete_waveform
+~~~~~~~~~~~~~~~
 
     .. py:currentmodule:: nifgen.Session
 
-    .. py:method:: delete_script(script_name)
+    .. py:method:: delete_waveform(waveform_name_or_handle)
 
-            Deletes the specified script from onboard memory.
+            Removes a previously created arbitrary waveform from the signal generator memory and invalidates the waveform handle.
```

Comment:
`and invalidates the waveform handle` only applies when a handle is used to call. I think you can just remove this part of the sentence.